<launch>
	<group ns="path_follower">
		<node pkg="path_follower" type="path_follower_node.py" 
			name="path_follower_node" output="screen"
			>
            <param name="robot_frame" value="base_link" />
            <param name="position_controller_nav_frame" value="world" />
            <param name="pose_threshold_d" value="0.5" />
            <param name="pose_threshold_r" value="0.2" />
		</node>
	</group>
</launch>